src/test/ui/type-mismatch.stderr

```diff
@@ -194,16 +194,16 @@ LL |     want::<Foo<foo, B>>(f); //~ ERROR mismatched types
    |                         ^ expected struct `B`, found struct `A`
```

Comment:
Note this error becomes awkward:
```
                         ^ expected struct `B`, found struct `A`

    = note: expected type `Foo<_, B>`
              found type `Foo<_, A>`
              found type `Foo<_>`
```

Maybe this particular case should display the parameters.